pass.cpp

```diff
@@ -125,7 +125,9 @@ void Pass::run(const Plasma::RunnerContext &context, const Plasma::QueryMatch &m
 {
     Q_UNUSED(context);
 
-    auto ret = QProcess::execute(QString("pass -c ") + match.text());
+    auto ret = match.text().contains("totp-", Qt::CaseInsensitive) ?
```

Comment:
Can you split the the path into its parts and check if any part starts with `totp-`? Thanks for the contribution!